ui/src/shared/components/DeleteDataForm/BucketsDropdown.tsx

```diff
@@ -22,7 +22,22 @@ const BucketsDropdown: FunctionComponent<Props> = ({
   bucketName,
   onSetBucketName,
 }) => {
-  const bucketNames = buckets.map(bucket => bucket.name)
+  const handleDefaults = name => name === '_monitoring' || name === '_tasks'
```

Comment:
This is inside a function component which renders on every frame, roughly 60 times a second. This function definition gets called every time `render` gets called, but it's not using any of the information passed in to render. It's basically needlessly being redefined - it can safely be defined outside the function once and just called every frame. 

Was this line run through prettier? This line, at a glance, might be mistaken for a greater than or less than comparison ("is this a function, or is checking if `name` is greater than or equal to `name` equaling `'_monitoring'` or `'_tasks'`?". It might help readability / comprehension to put parens around the argument and braces and a return statement around the body.

Reply:
Great suggestion about moving it outside of the FC. Prettier removed the `()` :(